src/seedu/addressbook/commands/Tagging.java

```diff
@@ -0,0 +1,23 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.data.tag.Tag;
+
+public class Tagging {
```

Comment:
I can't agree that this is an Association class. Why? You are not maintaining the Person and the Tag instances which this Tagging class is suppose to keep track of. 

Hints: 
Get a constructor as public Tagging(boolean tagOperation, Person person, Tag tag)
Maintain private instance variables of type Person, Tag, TagOperationand instantiate them in the constructor
Provider getters for these variables.